### PR TITLE
itch.io parser: Adding support for install drive redirect when parsing Windows games on Linux

### PR DESCRIPTION
**Pull request by Apalatn**
Just making it possible to use SRM on a Steam Deck with fewer problems by letting users replace the Windows install drive letter with a path to a virtual install drive folder.

(I'm not very familiar with typescript/javascript/node.js, so please pardon any weird mistakes I might have made.)

### CODE REVIEW

File: src/lib/parsers/itch-io.parser.ts
Comment (cbartondock) [inline src/lib/parsers/itch-io.parser.ts:1]: Why not use `fs.lstatSync(input).isDirectory()` instead of `!fs.lstatSync(input).isFile()`

**Comment by cbartondock** [comment]
Can you give me an example of exactly how this works? I'm not sure I see what exactly is new (ie that the appdata override path couldn't already do).

**Comment by Apalatn** [comment]
@cbartondock The issue is that if you install the windows version of the itch app on the Steam Deck, then use SRM to parse and add its games to steam, all of the paths in the steam shortcuts are windows paths instead of the correct Proton prefix (I think that's the right word?) paths.

Basically, this:
Screenshot_20230318_181042

This change allows you to substitute the correct proton path prefix for the windows one. For example, I use "/home/deck/.local/share/Steam/steamapps/compatdata/2475053656/pfx/drive_c" which correctly redirects to the virtual Proton C drive in the steam shortcut:

Screenshot_20230318_182735

File: src/lib/parsers/itch-io.parser.ts
Comment (Apalatn) [reply src/lib/parsers/itch-io.parser.ts:1]: I was just following the pattern already in the parser file. Changed.

**Comment by cbartondock** [comment]
Okay sounds good. We've changed the structure of the langauge data to facilitate localization through crowdin. If you wouldn't mind updating the PR so that it no longer conflicts with master I will merge it. If you want I can also do the merge.

**Pull request merged**